packages/core/EventManager.ts

```diff
@@ -454,7 +454,8 @@ export default class EventManager {
     // Because we are just cleaning up the events and meetings, we don't want to throw an error if one of them fails.
     (await Promise.allSettled(allPromises)).some((result) => {
       if (result.status === "rejected") {
-        log.error(
+        // Make it a soft error because in case a PENDING booking is rescheduled there would be no calendar events or video meetings.
```

Comment:
Don't really get this; log.error does not stop execution 😕

Reply:
It was just showing up as an error in logs, which is a false indicator. I converted into warn so that we know it could happen but still should be aware.